packages/ui-view/src/View/index.tsx

```diff
@@ -46,8 +46,6 @@ import type { ViewProps } from './props'
 ---
 category: components
 ---
-API:
-  - [View](https://instructure.design/#View)
 @module View
```

Comment:
While this is some random info here that is not needed, we should add here short descriptions in the future because they show up in the IDE when I hover over the component.